package.json

```diff
@@ -66,30 +66,30 @@
   },
   "homepage": "https://github.com/ReactiveX/RxJS",
   "devDependencies": {
-    "babel": "^5.8.23",
-    "babelify": "^6.1.2",
-    "benchmark": "^1.0.0",
+    "babel": "5.8.23",
+    "babelify": "6.1.2",
+    "benchmark": "1.0.0",
     "benchpress": "2.0.0-alpha.37.2",
-    "browserify": "^11.0.0",
-    "colors": "^1.1.2",
-    "coveralls": "^2.11.4",
-    "esdoc": "^0.2.5",
+    "browserify": "11.0.0",
+    "colors": "1.1.2",
+    "coveralls": "2.11.4",
+    "esdoc": "0.2.5",
     "eslint": "1.5.1",
-    "fs-extra": "^0.24.0",
-    "glob": "^5.0.14",
-    "google-closure-compiler": "^20150920.0.0",
-    "http-server": "^0.8.0",
-    "istanbul": "^0.3.22",
-    "jasmine": "^2.3.2",
-    "jasmine-core": "^2.2.0",
-    "lodash": "^3.5.0",
-    "platform": "^1.3.0",
-    "promise": "^7.0.3",
+    "fs-extra": "0.24.0",
+    "glob": "5.0.14",
+    "google-closure-compiler": "20150920.0.0",
+    "http-server": "0.8.0",
+    "istanbul": "0.3.22",
+    "jasmine": "2.3.2",
+    "jasmine-core": "2.2.0",
+    "lodash": "3.5.0",
+    "platform": "1.3.0",
+    "promise": "7.0.3",
     "protractor": "2.2.0",
     "rx": "latest",
```

Comment:
still picking rx as latest to be used in micro perf (micro perf test is only consumer for rx for now) for getting comparison number with latest. debatable though,  since had issues with regression recently(https://github.com/ReactiveX/RxJS/issues/501).